src/screens/Airdrop/ThankYouScreen.tsx

```diff
@@ -0,0 +1,96 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React, { Component } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+import { connect } from 'react-redux';
+
+import { ScreenTemplate, Button, Header } from 'app/components';
+import { Route } from 'app/consts';
+import { ApplicationState } from 'app/state';
+import { actions } from 'app/state/airdrop';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../loc');
+
+interface MapStateProps {
+  thankYouSeen: boolean;
+}
+
+interface ActionProps {
+  markThankYouSeen: Function;
```

Comment:
same as Pawel mentioned before - simply to not forget 🙂

Reply:
sure, thanks :)

Reply:
done